Python/ceval.c

```diff
@@ -4290,7 +4303,8 @@ _PyEval_EvalCode(PyThreadState *tstate,
             missing++;
         }
         if (missing) {
-            missing_arguments(tstate, co, missing, -1, fastlocals, qualname);
+            missing_arguments(tstate, co, missing, -1, fastlocals,
+                              qualname);
```

Comment:
Out of curiosity, why are you making these cosmetic changes? Is it just to make it show up in the diff so people can see what function calls are affected?

Reply:
This change is to fit into 80 columns :-)

Reply:
Oh, okay. I couldn't tell those lines were over by looking.